dsf-bpe/dsf-bpe-server/src/main/java/org/highmed/dsf/bpe/listener/StartListener.java

```diff
@@ -0,0 +1,31 @@
+package org.highmed.dsf.bpe.listener;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.ExecutionListener;
+import org.highmed.dsf.bpe.Constants;
+import org.highmed.dsf.fhir.variables.OutputWrapper;
+import org.hl7.fhir.r4.model.Task;
+
+public class StartListener implements ExecutionListener
```

Comment:
Consider adding a few lines of documentation, explaining the purpose of this class within the framework.